src/cloudwatch/modules/configuration/metadatareader.py

```diff
@@ -20,12 +20,15 @@ class MetadataReader(object):
     _CONNECT_TIMEOUT_IN_SECONDS = 0.3
     _RESPONSE_TIMEOUT_IN_SECONDS = 0.5
     _REQUEST_TIMEOUT = (_CONNECT_TIMEOUT_IN_SECONDS, _RESPONSE_TIMEOUT_IN_SECONDS)
+    _TOKEN_REQUEST = "latest/api/token"
+    _TOKEN_TTL_SECONDS = 21600 # 6 hours
```

Comment:
can we get the ttl from the request response body instead of hardcoding it? just in case the ttl would change in the future?

Reply:
the ttl is what we set to backend. the example of IMDSv2 is 6 hours. like 
https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/configuring-instance-metadata-service.html

```
The following example uses a Linux shell script and IMDSv2 to retrieve the top-level instance metadata items. The example command:

    Creates a session token lasting six hours (21,600 seconds) using the PUT request

    Stores the session token header in a variable named TOKEN

   ....


```